scripts/setup-ironic-networking.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- name: Add interfaces and bridges for ironic
+  hosts: localhost
+  tasks:
+    - name: Add config in osa_interfaces.cfg.j2
+      blockinfile:
+        dest: /opt/rpc-openstack/openstack-ansible/tests/roles/bootstrap-host/templates/osa_interfaces.cfg.j2
```

Comment:
Instead of modifying the template, we could probably just create a new file in `/etc/network/interfaces.d/`. That way we can put these tasks in ironic-pre-config.yml instead of having to separate them so this runs before bootstrap-aio. This can be a task we do after we get a working deployment, though.